app/src/main/java/com/door43/translationstudio/core/TargetTranslationMigrator.java

```diff
@@ -351,48 +355,93 @@ private static boolean v2( File path) throws Exception {
         return true;
     }
 
-   /**
-    * Merges chunks found in the target translation projects that do not exist in the source translation
-    * to a sibling chunk so that no data is lost.
-    * @param library
-    * @param translationSlugs target translations to merge
-    * @return
-    */
-    public static boolean migrateChunkChanges(final Translator translator, final Library library, final String[] translationSlugs) {
-        boolean mergeSuccess = true;
-        for (String translationSlug : translationSlugs) {
-            final TargetTranslation targetTranslation = translator.getTargetTranslation(translationSlug);
-            boolean success = migrateChunkChanges(library, targetTranslation);
-            mergeSuccess = mergeSuccess && success;
+    /**
+     * convert project from USX to USFM
+     * @param manifest
+     * @param targetTranslationDir
+     * @return
+     */
+    private static boolean migrateFromUSXtoUSFM(final JSONObject manifest, final File targetTranslationDir) {
+
+        Translator translator = AppContext.getTranslator();
+        TargetTranslation targetTranslation = TargetTranslation.open(targetTranslationDir);
```

Comment:
We cannot depend on the target translation object here because the migrateFromUSXtoUSFM method called assuming the target translation is at v5 of the manifest. It is possible we will have to make changes in v6 or v7 that would break this.